Source/Core/Core/HW/EXI/BBA/TAPServer_Apple.cpp

```diff
@@ -0,0 +1,127 @@
+// Copyright 2020 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include <fcntl.h>
+#include <sys/socket.h>
+#include <sys/un.h>
+#include <unistd.h>
+
+#include "Common/Logging/Log.h"
+#include "Common/StringUtil.h"
+#include "Core/HW/EXI/EXI_Device.h"
+#include "Core/HW/EXI/EXI_DeviceEthernet.h"
+
+namespace ExpansionInterface
+{
+// This interface is only implemented on macOS, since macOS needs a replacement
+// for TunTap when the kernel extension is no longer supported. This interface
+// only appears in the menu on macOS, so on other platforms, it does nothing and
+// refuses to activate.
+
+static const std::string socket_path = "/tmp/dolphin-tap";
```

Comment:
```suggestion
constexpr char socket_path[] = "/tmp/dolphin-tap";
```

So that this doesn't require a runtime static constructor.